### DISCUSSION TITLE
Converting from imgui to imgui_bundle

### DISCUSSION THREAD

**Discussion by FaceCrap** [pthom/imgui_bundle#296] [Q&A]
I am working with a python app which currently is using [imgui](https://pypi.org/project/imgui/).
I would like to convert it to using imgui_bundle.
I did a quick trial replacing the module imports but am already running into issues with it not recognizing window and popup flags. e.g. `imgui.WINDOW_NO_MOVE`

Is there documentation someplace which describes what I need to replace with imgui_bundle specific statements/constants that don't require me to wade through countless header files or source code files?

**Comment by pthom** [comment]
Well, there is a complete set of documented stubs in Dear ImGui Bundle, they should be recognized by your IDE, and give you an autocompletion.

example:
<img width="690" alt="image" src="https://github.com/user-attachments/assets/09dfd376-4900-48a8-a85f-c6e23689ea89" />


And those stubs are also available online:

https://github.com/pthom/imgui_bundle/blob/5e3132343f66de16d5fa0540f94d1b3af1ee72b4/bindings/imgui_bundle/imgui/__init__.pyi#L3055-L3072
